newspaper/github.py

```diff
@@ -0,0 +1,164 @@
+            if "," in raw_total_stars:
+                raw_total_stars = raw_total_stars.replace(",", "")
+        if raw_total_stars:
+            total_stars: int
+            try:
+                total_stars = int(raw_total_stars)
+            except ValueError as missing_number:
+                print(missing_number)
+        else:
+            total_stars = None
+
+        # forks
+        if stars_built_section.a.findNextSibling("a"):
+            raw_forks = stars_built_section.a.findNextSibling(
+                "a",
+            ).get_text(strip=True)
+            if "," in raw_forks:
+                raw_forks = raw_forks.replace(",", "")
+        if raw_forks:
+            forks: int
+            try:
+                forks = int(raw_forks)
+            except ValueError as missing_number:
+                print(missing_number)
+        else:
+            forks = None
+
+        # stars in period
+        if stars_built_section.find(
+            "span",
+            class_="d-inline-block float-sm-right",
+        ):
+            raw_stars_since = (
+                stars_built_section.find(
+                    "span",
+                    class_="d-inline-block float-sm-right",
+                )
+                .get_text(strip=True)
+                .split()[0]
+            )
+            if "," in raw_stars_since:
+                raw_stars_since = raw_stars_since.replace(",", "")
+        if raw_stars_since:
+            stars_since: int
+            try:
+                stars_since = int(raw_stars_since)
+            except ValueError as missing_number:
+                print(missing_number)
+        else:
+            stars_since = None
+
+        # builtby
+        built_section = stars_built_section.find(
+            "span",
+            class_="d-inline-block mr-3",
+        )
+        if built_section:
+            contributors = stars_built_section.find(
+                "span",
+                class_="d-inline-block mr-3",
+            ).find_all("a")
+            built_by = []
+            for contributor in contributors:
+                contr_data = {}
+                contr_data["username"] = contributor["href"].strip("/")
+                contr_data["url"] = "https://github.com" + contributor["href"]
+                contr_data["avatar"] = contributor.img["src"]
+                built_by.append(dict(contr_data))
+
+        repositories = {
+            "rank": rank + 1,
+            "username": username,
+            "repositoryName": repository_name,
+            "whole_name": username + "/" + repository_name,
+            "url": repo_url,
+            "description": description,
+            "language": language,
+            "languageColor": lang_color,
+            "totalStars": total_stars,
+            "forks": forks,
+            "starsSince": stars_since,
+            "since": since,
+            # "builtBy": built_by, ## Not needed for now
```

Comment:
```suggestion
            "builtBy": built_by,
```
I would probably keep this since we are still keeping the code that gets `built_by`.

Reply:
I've disabled because it makes the JSON that's returned a lot longer (and more unreadable) and we don't need it